### DISCUSSION TITLE
No affiliation

### DISCUSSION THREAD

**Discussion by castedo** [tskit-dev/msprime-1.0-paper#268] [General]
The journal is complaining that "per journal style, affiliation is mandatory for all authors" in reference to the "No affiliation" note for @hugovk.

IMO the preprint is 100% **accurate** and **transparent** about the lack of affiliation for @hugovk. Arguably I should be doing the same as @hugovk.

I go into more details on my opinions on accuracy and transparency in #266.

Last time I published in a journal (with no affiliation) I simply listed my home address (https://doi.org/10.1016/j.ic.2006.10.007). Not that this is clearly relevant here, but if the journal feels the need to have postal addresses, you could just put a postal address. The deeper question is what's the point of these author notes? To disambiguate common names? To be able to contact authors (via postal mail)? To show off? To appear more legitimate?

**Comment by petrelharp** [comment]
I think a good way to interpret this sort of thing is that it's *often* a good requirement - most authors are in academic departments with postal addresses, which they should put down - and in the cases where it's not a good requirement, journals are in my experience happy to do something different. Maybe I'm wrong? As for whether you should have an affiliation or not, that's totally up to you, I think - I could think of good reasons either way, but what matters most is that you're comfortable with it. Those affiliations serve lots of different purposes, as you say.

**Comment by jeromekelleher** [comment]
Under my interpretation we can still use "No affiliation" for @hugovk if we wish. I vote we do whatever @hugovk prefers, or, if he has no preference, whatever is simpler.

I agree that much of the affiliation thing is silly (I mean postal addresses, come on), but we're not going to fix the system here and there's no point in wasting time arguing about precisely how it's broken.

**Reply by hugovk** [reply]
I suggested something like "Independent researcher" in the email thread and the editorial office confirmed:

> For the affiliation question -- Independent Researcher is appropriate and what we've used in the past.

So that's fine by me! Here's a PR: https://github.com/tskit-dev/msprime-1.0-paper/pull/269
